charts/nginz/templates/conf/_nginx.conf.tpl

```diff
@@ -343,10 +342,8 @@ http {
         }
         more_set_headers 'Access-Control-Allow-Origin: $http_origin';
     }
-    {{ end }}
 
     # Swagger UI
-
     location /swagger-ui {
```

Comment:
Now this block is under `{{ if not (eq $.Values.nginx_conf.env "prod")  }}` as well.